ui/src/dashboards/utils/tempVars.ts

```diff
@@ -59,18 +59,18 @@ const reconcileTempVarsWithOverrides = (
       const overriddenValues = values.map(tempVarValue => {
         const {value} = tempVarValue
         if (value === overrideValue) {
-          return {...tempVarValue, picked: true}
+          return {...tempVarValue, localSelected: true}
         }
-        return {...tempVarValue, picked: false}
+        return {...tempVarValue, localSelected: false}
       })
       return {...tempVar, values: overriddenValues}
     } else {
       // or pick selected value.
```

Comment:
could prob remove this comment -- if its meaning is not captured in this block, maybe a var name could be changed? ideally the code is self-documenting, and i'm not clear how this one adds to what i can see just from var names and logic

Reply:
Jared, these are just overlooked and leftover from pseudocoding.